Easydict/NewApp/Utility/Extensions/QueryService+ConfigurableService/DeepLTranslate+ConfigurableService.swift

```diff
@@ -15,6 +16,26 @@ enum DeepLAPIUsagePriority: String, CaseIterable, Hashable {
     case authKeyOnly = "2"
 }
 
+extension DeepLAPIUsagePriority: Defaults.Serializable {
```

Comment:
An `enum` with `rawValue` is already comfort to `Defaults.Serializable`. We can just use a line

```swift
extension DeepLAPIUsagePriority: Defaults.Serializable {}
```

Reply:
OK, removed.